backend/cmm.ml

```diff
@@ -156,7 +156,7 @@ type rec_flag = Nonrecursive | Recursive
 
 type prefetch_temporal_locality_hint = Nonlocal | Low | Moderate | High
 
-type atomic_op = Fetch_and_add | Compare_and_swap
+type atomic_op = Fetch_and_add | Compare_and_swap | Exchange | Compare_exchange
```

Comment:
Can we rename `Compare_and_swap` to `Compare_and_set` so it matches the stdlib? It's not clear what the difference is vs. `Compare_exchange`

Reply:
I'll make separate PR for the change.